modules/layer1.js

```diff
@@ -0,0 +1,17 @@
+var Layer1 = `
```

Comment:
What does `Layer1` mean?

Treat your code as if it were a book. I should be able to understand what your modules do by just looking at their names.

Reply:
okay sir.